src/rtextures.c

```diff
@@ -187,6 +187,13 @@
     #include "external/stb_image_resize.h"  // Required for: stbir_resize_uint8() [ImageResize()]
 #endif
 
+#define NANOSVG_IMPLEMENTATION	// Expands implementation
```

Comment:
It requires a flag in `config.h` for `SUPPORT_FILEFORMAT_SVG` and the corresponding checks to support this functionality.

Reply:
I've added the flag in `config.h` and put it around the NanoSVG headers.